Sources/Segment/ObjC/ObjCAnalytics.swift

```diff
@@ -63,8 +63,16 @@ extension ObjCAnalytics {
     ///   - traits: A dictionary of traits you know about the user. Things like: email, name, plan, etc.
     /// In the case when user logs out, make sure to call ``reset()`` to clear user's identity info.
     @objc(identify:traits:)
-    public func identify(userId: String, traits: [String: Any]?) {
-        analytics.identify(userId: userId, traits: traits)
+    public func identify(userId: String?, traits: [String: Any]?) {
+        if let userId = userId {
+            analytics.identify(userId: userId, traits: traits)
```

Comment:
This looks like a recursive infinite loop? 

On line 68: is userId no an Optional anymore? and is there identify that doesn't take an optional?

Reply:
This is calling into the swift version of identify where userId is not optional.